bundles/org.openhab.binding.tplinksmarthome/README.md

```diff
@@ -241,33 +254,44 @@ Either `deviceId` or `ipAddress` must be set.
-|------------------|-----------|----------------------------------------------------|-----------------------------------------------------------------|
-| switch           | Switch    | Switch the Smart Home device on or off.            | HS100, HS103, HS105, HS110, HS200, HS210, KP100, RE270K, RE370K |
-| brightness       | Dimmer    | Set the brightness of Smart Home device or dimmer. | HS220, KB100, KL110, KL120, LB100, LB110, LB120, LB200          |
-| colorTemperature | Dimmer    | Set the color temperature of Smart Home light.     | KB130, KL120, KL130, LB120, LB130, LB230                        |
-| color            | Color     | Set the color of the Smart Home light.             | KB130, KL130, LB130, LB230                                      |
-| power            | Number    | Actual energy usage in Watt.                       | HS110, KLxxx, LBxxx                                             |
-| eneryUsage       | Number    | Energy Usage in kWh.                               | HS110                                                           |
-| current          | Number    | Actual current usage in Ampere.                    | HS110                                                           |
-| voltage          | Number    | Actual voltage usage in Volt.                      | HS110                                                           |
-| led              | Switch    | Switch the status led on the device on or off.     | HS100, HS103, HS105, HS110, HS200, HS210, HS220, KP100          |
-| rssi             | Number    | Wi-Fi signal strength indicator in dBm.            | All                                                             |
+| Channel Type ID  | Item Type | Description                                        | Thing types supporting this channel                                                         |
+|------------------|-----------|----------------------------------------------------|---------------------------------------------------------------------------------------------|
+| switch           | Switch    | Switch the Smart Home device on or off.            | HS100, HS103, HS105, HS107, HS110, HS200, HS210, HS300, KP100, KP200, KP400, RE270K, RE370K |
+| brightness       | Dimmer    | Set the brightness of Smart Home device or dimmer. | HS220, KB100, KL110, KL120, LB100, LB110, LB120, LB200                                      |
+| colorTemperature | Dimmer    | Set the color temperature of Smart Home light.     | KB130, KL120, KL130, LB120, LB130, LB230                                                    |
+| color            | Color     | Set the color of the Smart Home light.             | KB130, KL130, LB130, LB230                                                                  |
+| power            | Number    | Actual energy usage in Watt.                       | HS110, HS300, KLxxx, LBxxx                                                                  |
```

Comment:
A second question: Why don't you use UoM here. Are there plans to introduce it for this binding.

Reply:
I was wondering about that too recently :smile: The original implementation (for HS110) is from before UoM and the others just reuse the code. I was thinking about making them UoM, although it's also a breaking change for the HS110. But I was planning to change it after this pr was being merged.